CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ### Features
 
+- [#8042](https://github.com/influxdata/influxdb/issues/8042): Add bitwise AND, OR and XOR operators to the query language
```

Comment:
Minor nitpick. Can you move this to the bottom after you rebase? This is really annoying me for some reason, but it really shouldn't annoy me.